packages/angular/cli/commands/add.json

```diff
@@ -30,6 +30,11 @@
               "format": "hostname"
             }
           ]
+        },
+        "verbose": {
+          "description": "When true, display additional details during the add process.",
```

Comment:
`When true,` doesn't really make sense here since it's a flag.
Can you also make the description more generic?  I'd like to eventually make it a global option for all commands.
Something maybe like: `Display additional details about internal operations during execution.`

Reply:
I do agree with you, through probably regarding the “When true”, we should probably remove it from all of the other descriptions as well. As it seems to be some the standard for Boolean options.